framework/src/modules/chain/rounds/round.js

```diff
@@ -303,31 +303,27 @@ class Round {
 		if (exceptions.rounds[this.scope.round.toString()]) {
 			// Apply rewards factor
 			roundRewards.forEach((reward, subIndex) => {
-				roundRewards[subIndex] = new Bignum(reward.toPrecision(15))
-					.multipliedBy(
-						exceptions.rounds[this.scope.round.toString()].rewards_factor
-					)
-					.integerValue(Bignum.ROUND_FLOOR);
+				roundRewards[subIndex] = new BigNum(reward.toPrecision(15))
+					.times(exceptions.rounds[this.scope.round.toString()].rewards_factor)
+					.floor(BigNum.ROUND_FLOOR);
 			});
 
 			// Apply fees factor and bonus
-			roundFees = new Bignum(roundFees.toPrecision(15))
-				.multipliedBy(
-					exceptions.rounds[this.scope.round.toString()].fees_factor
-				)
+			roundFees = new BigNum(roundFees.toPrecision(15))
+				.times(exceptions.rounds[this.scope.round.toString()].fees_factor)
 				.plus(exceptions.rounds[this.scope.round.toString()].fees_bonus)
-				.integerValue(Bignum.ROUND_FLOOR);
+				.floor(BigNum.ROUND_FLOOR);
 		}
 
-		const fees = new Bignum(roundFees.toPrecision(15))
+		const fees = new BigNum(roundFees.toPrecision(15))
 			.dividedBy(ACTIVE_DELEGATES)
-			.integerValue(Bignum.ROUND_FLOOR);
-		const feesRemaining = new Bignum(roundFees.toPrecision(15)).minus(
-			fees.multipliedBy(ACTIVE_DELEGATES)
+			.floor(BigNum.ROUND_FLOOR);
+		const feesRemaining = new BigNum(roundFees.toPrecision(15)).minus(
+			fees.times(ACTIVE_DELEGATES)
 		);
 		const rewards =
-			new Bignum(roundRewards[index].toPrecision(15)).integerValue(
-				Bignum.ROUND_FLOOR
+			new BigNum(roundRewards[index].toPrecision(15)).floor(
```

Comment:
I think `floor` doesn't take the input in the `@liskhq/bignum`

Reply:
As we checked the parameter was inert but removed for clarity of code.